Readme.md

```diff
@@ -10,34 +10,34 @@ This example demonstrates various techniques that allow you to bind the [GridCon
 
 ## The Task
 
-GridControl is a data-aware control. It displays data from a source collection. The data is usually pre-determined and can be easily obtained with property descriptors for a class that represents source records.
+The GridControl is a data-aware control that displays data from a source collection. In most cases, this data is predetermined, and the GridControl uses its property descriptors to obtain data from a data source.
 
-This data structure does not always satisfy the developer needs. One of the possible requirements is that values should be calculated at runtime and submitted into a collection of an arbitrary size. It may not always be possible to use predefined accessors for such values.
+Sometimes, data source values are calculated at runtime and placed into a collection of an arbitrary size. In this collection, not only the number of rows but also the number of columns can change. You cannot use predefined accessors for such values.
 
 
 ## The Standard Technique
 
-The standard WPF data binding solves this problem. Provided the correct path, the data will be displayed anywhere you need it. In fact, you can also use it with GridControl - see [Binding Columns to Data Source Fields](https://docs.devexpress.com/WPF/120400/controls-and-libraries/data-grid/grid-view-data-layout/columns-and-card-fields/binding-columns-to-data-source-fields). This technique has its flaws. In certain cases and with excessive amounts of data, bindings may show significant performance issues.
+To accomplish this task, you can use the standard WPF data binding. Refer to the following help for information on how to use data bindings in the GridControl: [Binding Columns to Data Source Fields](https://docs.devexpress.com/WPF/120400/controls-and-libraries/data-grid/grid-view-data-layout/columns-and-card-fields/binding-columns-to-data-source-fields).
 
-We recommend that you use bindings for columns only when it is necessary. For example, to display unsupported data formats with custom cell templates or to make GridControl work with interface inheritance.
+This technique has its flaws. In certain cases and with excessive amounts of data, bindings may show significant performance issues. We recommend that you use bindings for columns only when it is necessary. For example, to display unsupported data formats with custom cell templates or to make the GridControl work with the interface inheritance.

```

Comment:
This sentence has a couple of issues:

- `has its flaws` This is a strange phrase for a technical doc
- `with excessive amounts of data`  What would be an excessive amount? This is why it is best to avoid adverbs and adjectives since there is no clear indication of how much data would be 'excessive'. And since 'excessive' means more than necessary, normal, or desirable, what would be a necessary, normal, or desirable amount of data?
- `show significant performance issues` Again, another adjective - what would you consider as a significant issue?

Reply:
```suggestion
In certain cases and with a large amount of data, bindings may cause performance issues. We recommend that you use bindings for columns only when it is necessary. For example, to display unsupported data formats with custom cell templates or to make the GridControl work with the interface inheritance.
```